recipes/dagmc/meta.yaml

```diff
@@ -29,6 +29,8 @@ requirements:
     - python
     - hdf5
     - moab 5.*
+    - libblas
```

Comment:
What do we need BLAS for?

Reply:
I think that was the wrong one. I need libopenblas